src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/stores/FormStore.js

```diff
@@ -129,6 +133,6 @@ export default class FormStore {
         }
 
         this.type = type;
-        this.resourceStore.set(TYPE, type);
+        this.set(TYPE, type);
```

Comment:
This means changing the template also doesn't trigger the save button, right? I don't think that this is the desired behavior... This should be only like that initially. Maybe also differ between `setType` and `changeType`?